homeassistant/components/xiaomi_aqara/manifest.json

```diff
@@ -3,8 +3,17 @@
   "name": "Xiaomi Gateway (Aqara)",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_aqara",
-  "requirements": ["PyXiaomiGateway==0.13.2"],
-  "after_dependencies": ["discovery"],
-  "codeowners": ["@danielhiversen", "@syssi"],
-  "zeroconf": ["_miio._udp.local."]
-}
+  "requirements": [
+    "PyXiaomiGateway==0.13.3"
```

Comment:
Please don't change the format and bump the version number only.

Reply:
Sorry for the format change :( That must be my IDE that changed it on save. Thanks for rectifying the mistake